include/feedcontainer.h

```diff
@@ -43,7 +43,7 @@ class FeedContainer {
 	std::vector<std::shared_ptr<RssFeed>> feeds;
```

Comment:
This variable is still public so code outside of this class can still use/change the feeds vector without locking `FeedContainer::feeds_mutex`.

The only classes directly using this variable are the `opml` and `Controller` classes:
- `opml`: Probably is not an issue as it is only used for exporting (looks like this is always done before/without starting extra threads so should be save)
- `Controller`: I think the uses of `feedcontainer.feeds` might allow some race conditions. When accessing the `feeds` variable, it locks `Controller::feeds_mutex` but this is not the same mutex as the mutex used in the FeedContainer class.

Reply:
@Minoru: Thanks for opening the issue. That resolves this comment.